src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -426,7 +427,27 @@ PerformAudioPassThruRequest::PrepareAudioPassThruResultCodeForResponse(
       (tts_result != hmi_apis::Common_Result::INVALID_ENUM)) {
     common_result = hmi_apis::Common_Result::WARNINGS;
     out_result = true;
-  } else if (ui_result == hmi_apis::Common_Result::INVALID_ENUM) {
+  }
+
+  else if ((ui_result == hmi_apis::Common_Result::WARNINGS ||
+            tts_result == hmi_apis::Common_Result::WARNINGS) &&
+           Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
```

Comment:
@VVeremjova I think you can use is_ok flag of the ResponseInfo structure instead of the Compare<>. It is set in ResponseInfo constructor. It will make this statement much more readable.